homeassistant/components/open_meteo/weather.py

```diff
@@ -33,7 +37,9 @@ class OpenMeteoWeatherEntity(
 ):
     """Defines an Open-Meteo weather entity."""
 
-    _attr_temperature_unit = TEMP_CELSIUS
+    _attr_native_precipitation_unit = LENGTH_MILLIMETERS
+    _attr_native_temperature_unit = TEMP_CELSIUS
+    _attr_native_wind_speed_unit = SPEED_KILOMETERS_PER_HOUR
```

Comment:
I did not find documentation about what unit the open_meteo package uses for precipitation